tools/checkTheme.js

```diff
@@ -193,7 +193,7 @@ class CSSRule {
   constructor(rule, body) {
     this.rule = rule;
     if (rule.declarations) {
-      this.bg = rule.declarations.find(x => x.property =="background")?.value;
+      this.bg = rule.declarations.find(x => x.property =="background" || x.property =="background-color")?.value;
```

Comment:
Nice, but I think actually we'd need two finds to be "correct", no?  It would be possible to have:

```css
background:  content-box no-repeat something;
background-color: specific color;
```

So you'd want to first scan ALL the declarations for `background-color`, and then failing that scan a second pass for `background`?

Reply:
OK. Done.